src/main/scala/mastermind/Mastermind.scala

```diff
@@ -13,19 +13,36 @@ val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
 def getRandomColor(): Color =
-  ???
+  Random.shuffle(validColors).head
 
 /** Given four colors, make a board from them */
 def makeBoardFromColors(c1: Color, c2: Color, c3: Color, c4: Color): Board =
-  ???
+  List(c1, c2, c3, c4).mkString("")
```

Comment:
Found a way to do this that is slightly different from what you did. Instead of converting all of the colors to a string at once, I used .toString  to convert each color to a sting and then added them together: 
c1.toString + c2.toString + c3.toString + c4.toString